infra/src/main/java/com/redhat/service/smartevents/infra/api/models/responses/PagedListResponse.java

```diff
@@ -6,8 +6,11 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.redhat.service.smartevents.infra.models.ListResult;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(name = "List",
```

Comment:
Sort of the same question here

Reply:
@wtrocki am I correct here, for the `List`, is the answer same as for `ObjectReference`?

Reply:
yes